util/src/main/java/org/hyperledger/besu/util/Subscribers.java

```diff
@@ -37,8 +37,8 @@
  * subscribers.unsubscribe(this::onSomeEvent);</code>
  * </pre>
  *
- * <p>Since the two separate <code>this:onSomeEvent</code> are not equal, the subscriber wouldn't be
- * removed. This bug is avoided by assigning each subscriber a unique ID and using that to
+ * <p>Since the two separate <code>this::onSomeEvent</code> are not equal, the subscriber wouldn't
+ * be removed. This bug is avoided by assigning each subscriber a unique ID and using that to
```

Comment:
nit: maybe spotless messed with this file? doesn't look like it needs to change.

Reply:
I edited "this:onSomeEvent" -> "this::onSomeEvent" and then spotless rejigged it :)